python/stempy/io/__init__.py

```diff
@@ -85,3 +86,6 @@ def save_stem_images(outputFile, images, names):
         stem_group = f.require_group('stem')
         dataset = stem_group.create_dataset('images', data=images)
         dataset.attrs['names'] = names
+
+def reader_h5(h5dataset, imagewidth , imageheight , scanwidth, scanheight, blocksize=32, blockNumInFile=25, totalImgNum=25600):
```

Comment:
The image width and height can be got from the h5 data set.

Reply:
We should also check for the presents of `Nx` and `Ny` attributes, these would hold the scan with and height.

Reply:
I could update this part further.